runtime/src/dpe_platform.rs

```diff
@@ -34,6 +35,7 @@ pub struct DpePlatform<'a> {
     auto_init_locality: u32,
     hashed_rt_pub_key: Digest,
     cert_chain: &'a mut ArrayVec<u8, MAX_CERT_CHAIN_SIZE>,
+    image_header: ImageHeader,
```

Comment:
Can we pass this by reference to avoid copying every time?

Reply:
I was trying to do this earlier but it would take a large refactor since we have both mutable and immutable borrows of persistent data so the borrow checker complains. ImageHeader is only 150 bytes so it's not too big of a copy

Reply:
The option is we could extract each of the fields we need from the image header and pass it to the platform so we copy even less data.

Reply:
Refactored to only pass in the nb and nf validities so that we copy 30 bytes instead of 150.